modules/installation-creating-gcp-private-dns.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-user-infra-vpc.adoc
+
+[id="installation-creating-gcp-private-dns_{context}"]
+= Creating a private DNS zone in GCP
+
+You must configure a private DNS zone in Google Cloud Platform (GCP) for your
+{product-title} cluster to use. One way to create this component is
+to modify the provided Deployment Manager template.
+
+[NOTE]
+====
+If you do not use the provided Deployment Manager template to create your GCP
+infrastructure, you must review the provided information and manually create
+the infrastructure. If your cluster does not initialize correctly, you might
+have to contact Red Hat support with your installation logs.
+====
+
+.Prerequisites
+
+* Configure a GCP account.
+* Generate the Ignition config files for your cluster.
+* Create and configure a VPC and associated subnets in GCP.
+
+.Procedure
+
+. Copy the template from the *Deployment Manager template for the private DNS*
+section of this topic and save it as `02_infra.py` on your computer. This
```

Comment:
The file is named 02_infra.py but 02_dns.py is used in procedure 3. Please make them consistent.